### PR TITLE
added parameters

### LINKED ISSUE #2: add additional parameters for the various models
BlackForestLabs offers additional inputs for some of the models.

For example, `flux-pro-1.1` also includes: `prompt_upsampling`, `safety_tolerance`.

Can we have those added?

Here's the list of inputs per model:

https://api.bfl.ml/scalar#model/fluxdevinputs
https://api.bfl.ml/scalar#model/fluxpro11inputs
https://api.bfl.ml/scalar#model/fluxproinputs
https://api.bfl.ml/scalar#model/fluxultrainput

### LINKED ISSUE #5: A few parameters need to be moved/added
Currently `safety_tolerance` doesn't work on `flux-pro-1.1-ultra` and it should.

`flux-dev` add:
* `steps`
* `guidance`

`flux-pro` add:
* `steps`
* `guidance`
* `interval`

`flux-ultra` add:
* `raw`

### PR DESCRIPTION
**Pull request by shhlife**
Added docstring and additional parameters as per api specs.

Closes #2

### CODE REVIEW

File: Desktop/dog_skateboard_watercolor.jpeg
Comment (collindutter) [inline Desktop/dog_skateboard_watercolor.jpeg:1]: Did you mean to check this in? Cool dog tho.

**Comment by collindutter** [comment]
Does this close #2? If so, should update PR description with "Closes #2"

File: Desktop/dog_skateboard_watercolor.jpeg
Comment (shhlife) [reply Desktop/dog_skateboard_watercolor.jpeg:1]: nah, as I was testing it made a new dog & I thought.. glasses are cool. :) we can keep or drop, totally up to you!

Comment (collindutter) [reply Desktop/dog_skateboard_watercolor.jpeg:1]: Fine with keeping it but let's maybe move to an `assets/` directory and then reference it in the README. Right now it's in a directory called "Desktop" which doesn't quite fit the repo structure.

Comment (shhlife) [reply Desktop/dog_skateboard_watercolor.jpeg:1]: Okay, I can update the code to output to an assets dir

**Review by collindutter — APPROVED** [review]

**Pull request merged**